docs/content/docs/dev/datastream/fault-tolerance/serialization/types_serialization.md

```diff
@@ -167,6 +168,24 @@ input.keyBy(_.word)
 
 Flink supports all Java and Scala primitive types such as `Integer`, `String`, and `Double`.
 
+#### Common Collection Types
```

Comment:
I assume the sentence above "
Flink supports all Java and Scala primitive types such as `Integer`, `String`, and `Double`." is not true at flink 2 as it does not support Scala types.